### ISSUE TITLE
Ctrl + S does not save model

### ISSUE THREAD

**Issue by tkuchida** [opensim-org/opensim-gui#840]
Jen: Users expect common shortcuts from 3.3 to also work in 4.0

### Steps to reproduce
- open arm26 model
- in Navigator, select Bodies > base > TRIlongglen
- edit *translation* property (e.g., change the first number from -0.043905 to -0.1)
- press `<Ctrl> + S`
- File > Close Model

### Expected result
`<Ctrl> + S` saved the model. User should not be prompted to save changes because no changes were made since last save.

### Actual result
`<Ctrl> + S` did not save the model.

![](https://user-images.githubusercontent.com/4203505/41190624-6cc6164c-6b97-11e8-9481-1a5dd3cb0552.png)

### Environment and GUI version
AppVeyor artifact OpenSim-cedd8d22-2018-05-31-win64.exe

**Comment by tkuchida** [comment]
Similar issue encountered when opening and closing several models (without making changes):
1. open a model with Ctrl+O
2. inspect model
3. close model with Ctrl+F4
4. open another model with Ctrl+O

Nothing happens after step 4. The Visualizer Window has the focus; clicking in the (empty) Navigator pane returns the focus to the NetBeans GUI and Ctrl+O works again—but requiring an intermediate click defeats the purpose of the hotkey.

**Comment by aymanhab** [comment]
Build with the fix is available as artifact here 
https://ci.appveyor.com/project/opensim-org/opensim-gui/build/1.0.1605/artifacts

**Comment by chrisdembia** [comment]
I downloaded the aforementioned artifact and the original bug still exists: I am still prompted to save the model after typing `Ctrl + S` when focused in the Navigator.

**Comment by chrisdembia** [comment]
It's not clear to me why https://github.com/opensim-org/opensim-gui/pull/905 would fix this issue; my focus is not in the visualizer.

**Comment by aymanhab** [comment]
@chrisdembia The scenario described on June 14th by @tkuchida is a focus issue that should be resolved since Ctrl+S is not handled by the visualizer anymore and is handled by the GUI as shortcut. There could be another issue regarding how we recognize a model has changed that would be good to pinpoint if exists.

**Comment by aymanhab** [comment]
On further investigation, Ctrl+S is double used to save models and save scripts. I'll submit a PR to switch the latter to use ctrl-W. Doing the change locally fixes the Ctrl+S to save issue.

**Comment by aymanhab** [comment]
Needs verification in latest artifacts on both Windows, Mac. @tkuchida @chrisdembia @nickbianco

**Comment by nickbianco** [comment]
The shortcut also worked for me on Windows. I got the same dialog box and the change persisted after opening and closing the model.
